app/controllers/validations.js

```diff
@@ -0,0 +1,32 @@
+exports.validateUser = user => {
+  const errorMsgs = [];
+  const validMail = /((^[0-9]+[a-z]+)|(^[a-z]+[0-9]+))*@wolox.com.ar/;
+  const validPass = /((^[0-9]+[a-z]+)|(^[a-z]+[0-9]+))+[0-9a-z]+$/i;
```

Comment:
This regular expressions are wrong, check for example with a regex tester (example valid case `12345678a`). Do a more simple validation for the case maybe `([A-Za-z]+[0-9]+)|([0-9]+[A-Za-z]+)` for password validation